modules/data.land/R/IC_BADM_Utilities.R

```diff
@@ -0,0 +1,296 @@
+
+#Reading the data and writing the netcdf
+#' Read.IC.info.BADM
+#'
+#' @param lat numeric latitude
+#' @param long numeric longitude
+#'
+#' @description This function returns a dataframe of plant biomass, root and soil carbon for a set of lat and long coordinates.
+#' This function first finds the level1 and level2 ecoregions  for the given coordinates, and then tries to filter BADM database for those eco-regions. 
+#' If no data found in the BADM database for the given lat/longs eco-regions, then all the data in the database will be used to return the initial condition.
+#' All the variables are also converted to kg/m^2. 
+#' @return a dataframe with 8 columns of Site, Variable, Date, Organ, PlantWIni (Initial plant biomass, type of biomass can be found in the Variable and Organ), SoilIni (which shows the initial soil C), LitterIni, RootIni.
```

Comment:
Internal PEcAn functions should only return information in internal PEcAn standard variable names and units. https://pecanproject.github.io/pecan-documentation/develop/pecan-standard-formats.html

The returned object should also be plug-and-play with the existing IC workflow, which in this case probably means matching what comes out of the cohort2pool function and into the model-specific converter